.circleci/config.yml

```diff
@@ -0,0 +1,219 @@
+    resource_class: large
+    steps:
+      # Setup
+      - checkout
+      - attach_workspace:
+          at: *workspace
+      - restore-cache: *restore-cache-gradle
+
+      # Build app
+      - run:
+          name: Build exampleCustomVideoCapturer
+          command: ./gradlew -q exampleCustomVideoCapturer:assemble
+
+      # Save cache
+      - save-cache: *save-cache-gradle
+
+
+  build-exampleCustomVideoRenderer:
+    <<: *build-defaults
+    resource_class: large
+    steps:
+      # Setup
+      - checkout
+      - attach_workspace:
+          at: *workspace
+      - restore-cache: *restore-cache-gradle
+
+      # Build app
+      - run:
+          name: Build exampleCustomVideoRenderer
+          command: ./gradlew -q exampleCustomVideoRenderer:assemble
+
+      # Save cache
+      - save-cache: *save-cache-gradle
+
+  build-exampleDataTrack:
+    <<: *build-defaults
+    resource_class: large
+    steps:
+      # Setup
+      - checkout
+      - attach_workspace:
+          at: *workspace
+      - restore-cache: *restore-cache-gradle
+
+      # Build app
+      - run:
+          name: Build exampleDataTrack
+          command: ./gradlew -q exampleDataTrack:assemble
+
+      # Save cache
+      - save-cache: *save-cache-gradle
+
+
+  build-exampleScreenCapturer:
+    <<: *build-defaults
+    resource_class: large
+    steps:
+      # Setup
+      - checkout
+      - attach_workspace:
+          at: *workspace
+      - restore-cache: *restore-cache-gradle
+
+      # Build app
+      - run:
+          name: Build exampleScreenCapturer
+          command: ./gradlew -q exampleScreenCapturer:assemble
+
+      # Save cache
+      - save-cache: *save-cache-gradle
+
+  build-exampleVideoInvite:
+    <<: *build-defaults
+    resource_class: large
+    steps:
+      # Setup
+      - checkout
+      - attach_workspace:
+          at: *workspace
+      - restore-cache: *restore-cache-gradle
+
+      # Build app
+      - run:
+          name: Build exampleVideoInvite
+          command: ./gradlew -q exampleVideoInvite:assemble
+
+      # Save cache
+      - save-cache: *save-cache-gradle
+
+  build-quickstart:
+    <<: *build-defaults
+    resource_class: large
+    steps:
+      # Setup
+      - checkout
+      - attach_workspace:
+          at: *workspace
+      - restore-cache: *restore-cache-gradle
+
+      # Build app
+      - run:
+          name: Build quickstart
+          command: ./gradlew -q quickstart:assemble
+
+      # Save cache
+      - save-cache: *save-cache-gradle
+
+
+  build-quickstartKotlin:
+    <<: *build-defaults
+    resource_class: large
+    steps:
+      # Setup
+      - checkout
+      - attach_workspace:
+          at: *workspace
+      - restore-cache: *restore-cache-gradle
+
+      # Build app
+      - run:
+          name: Build quickstartKotlin
+          command: ./gradlew -q quickstartKotlin:assemble
+
+      # Save cache
+      - save-cache: *save-cache-gradle
+
+workflows:
+  version: 2
+
+  build-test:
```

Comment:
Sorry missed this, but let's rename from `build-test` to `build` ...There is no testing happening either.